### ISSUE TITLE
CSS styling

### ISSUE THREAD

**Issue by ghost** [Repl-it-Coders/Repl.it-2020#16]
It seems that the CSS in the website represents brutalism, in which can cause eyestrain (contrast... #fff and #000... yada yada)
This is not recommended under normal use. I have a Framework in development that should allow theming as well as Jamstacking.

After this, i have a plan to create a blog/forum template that is fully built with noSQL along side others.

I am available to help but overhauling the CSS takes **a lot of time**.

> anti-edit society

You must convert this from a format to text and then use 2 cyphers in the correct order.  One was used in an American Civil War with the codename "manchester" and the other was used by a person that was killed in 49AD.

**Comment by whippingdot** [comment]
I have no idea what you are saying...

Why can't you just edit the style.css file?

Also, is this just an ad for your upcoming framework?

Thanks for creating this issue anyway, I always wanted to know what 'brutalism' was lol

But actually, thanks for creating this issue if this 'brutalism' is actually bad. I will try to fix it with you.

**Comment by ghost** [comment]
@whippingdot no, im saying im planning to use it, HOWEVER, i will not apply it until the CSS framework is ready as... i dont like posting incomplete projects.

Brutalism means when you combine to absolute contrasting colors like pure white and black, blue and yellow ect ect. try to keep the contrast down.

Also, i work with Vandesm quite a bit, like getting electron.js, nw.js and neutrino to run in repl.it. :D

**EDIT: Sorry if this sounds like some weird flex.**

**Comment by firefish111** [comment]
why did you put some random cipher

**Comment by ghost** [comment]
@firefish111 because... i can

**Comment by whippingdot** [comment]
lul ROASTED

jk

Also, if you want to, you can help. Do you want to join the org? It is kinda as stale as a potato rn but it could get spicy 🌶🌶🌶🌶🌶 lul i am just copying what someone said

**Comment by iop3** [comment]
> lul i am just copying what someone said

who's the someone >:)

**Comment by whippingdot** [comment]
You, wasn't it?

**Comment by firefish111** [comment]
well that's called plagiarism

**Comment by iop3** [comment]
> You, wasn't it?

that was a joke. i dont think you got it lol. and yes it was me.

>  imma spice it up a bit. 🌶🌶🌶🌶🌶🌶🌶🌶🌶🌶🌶🌶🌶🌶

-TheQuackyDuck 2021

> well that's called plagiarism

😆 indeed.

**Comment by firefish111** [comment]
hehe time to put @whippingdot in jail

**Comment by whippingdot** [comment]
NO PLEASE

I AM JUST A LONELY DOT / PIXEL WHIPPING HIS WAY OUT OF PROBLEMS

~~i will whip you too mUhAhAHaAHahHAhahahHAhahHH~~

**Comment by firefish111** [comment]
and I am a fish full of fire, who needs matches when you have me

**Comment by whippingdot** [comment]
?

**Comment by firefish111** [comment]
> nevermind, just ignore this idiot named firefish
>       - god

**Comment by whippingdot** [comment]
Umderstood, oh above one

**Comment by iop3** [comment]
> nevermind, just ignore this idiot named firefish

😏

**Comment by firefish111** [comment]
@TheQuackyDuck rolj¶←ĸˀ¶ŧĸø←ˀøe¶ŧˀ←ŧ→←ˀØ¥Ø¥˝ı®øŧ´←→¶←¶eþ←→¶ŧ←→¶ŧ→←®E¥ ̛®˝&¥ ̛®&E←ˀ¶ŧĸyˀ¶ełˀø→¶ˀ↓←→ø¶ŧn¢ðɲŋđɲħðđ←nˀĸßðĸłðµßđĸłµðđĸłµĸ←ˀŧø←ˀŧßÐŊª&Ħ ̛ÐŊª& ̛Ð&GFKDFGDKFDFGKHJKDGFjhkdfgjklhjkdfŋħĸłðŋđˀħłĸŋđˀðĸłħĸðŧŋˀħĸłˀħðªÐ&ĦÐª ̛Ł&ÐĦ ̛Ħ&ŁÐ ̛ħˀðđĸħðˀđŋłħn←µe¶n─µn@łµe─@n½µ@¶ełµŧn¶łeŧẻ¶øŧĸĸłe¶ˀŧĸłe¶ˀłŧĸłˀŋħn→ˀðŋđħˀ¶ŧĸ←µĸ←n←nn←n↓ym,n↓ŧ─µn↓ŧ─

**Comment by ghost** [comment]
Yes ignore him jk jk... and spamming should also be ignored too. Hence why I deleted the message as it was literally just random unicode text.

**Comment by whippingdot** [comment]
samart stringant duv

**Comment by ghost** [comment]
Also, guys, you can see my, and @zplusfour and Vandesm11's progress of the new Framework at [Medusa's test page.](https://medusa.stringentdev.repl.co)

Things are slowly coming together. When this is done, we will use a Components system like react to create pages.

**Comment by firefish111** [comment]
@StringentDev goodbye from org

**Comment by ghost** [comment]
@firefish111 wait wut

**Comment by ghost** [comment]
@firefish111 you do know that authors of threads can delete messages right even if they are not part of the team?
And i can still comment here too.

**Comment by ghost** [comment]
@whippingdot 
> lul ROASTED
> 
> jk
> 
> Also, if you want to, you can help. Do you want to join the org? It is kinda as stale as a potato rn but it could get spicy 🌶🌶🌶🌶🌶 lul i am just copying what someone said

i mean like sure... Nando's chicken wing spicy???
If so im in.

**Comment by firefish111** [comment]
> @whippingdot
> 
> > lul ROASTED
> > jk
> > Also, if you want to, you can help. Do you want to join the org? It is kinda as stale as a potato rn but it could get spicy 🌶🌶🌶🌶🌶 lul i am just copying what someone said
> 
> i mean like sure... Nando's chicken wing spicy???
> If so im in.

In what? The void of organisationless emptiness?

**Comment by ghost** [comment]
Yes... yes really.

**Comment by iop3** [comment]
> Nando's chicken wing spicy???

?

sweetcsdesigns.com:

> Ingredients.
> - 1 tbsp extra virgin olive oil.
> - 12 chicken wings.
> - 1 tsp salt.
> - 1 tsp fresh cracked pepper.
> - 4 tbsp nandos sauce.
> - 2 tbsp butter.

**Comment by ghost** [comment]
@TheQuackyDuck YES, you are amazing for that comment.

**Comment by iop3** [comment]
> @TheQuackyDuck YES, you are amazing for that comment.

thank you xD

**Comment by whippingdot** [comment]
@StringentDev nandos got closed where my cousins live. they said they used to go every week :(

**Comment by whippingdot** [comment]
Oh you are already in the org...

**Comment by iop3** [comment]
this comment has been open for more than a week, closing due to idk.

**Comment by firefish111** [comment]
> > Apparently, I think it is funny to put a cipher, and watch these gullible idiots just like myself suffer, because I have the sense of humour of a two year old.

woah

**Comment by whippingdot** [comment]
?? where did you get that comment from...

**Comment by ghost** [comment]
@whippingdot that was firefish111 comment. He deleted it hence why this is shown:
![image](https://user-images.githubusercontent.com/80392427/125075720-e83e5180-e0b6-11eb-806e-a2a27adab53c.png)

**Comment by whippingdot** [comment]
for me it shows
![image](https://user-images.githubusercontent.com/64354499/125080589-a0352400-e0e2-11eb-9b73-ce264f4de081.png)

**Comment by ghost** [comment]
@whippingdot It is not like i can see as Firefish banned me. Thats probs his spam comment. WAIT hold up, i think that's mocking me. Hence Cipher and all.

**Comment by ghost** [comment]
@whippingdot got him, he edited my post, thats going in the report for him too.
![Screenshot_20210709-163352_Samsung Internet](https://user-images.githubusercontent.com/80392427/125102857-a5d73d80-e0d3-11eb-8087-d20df0aee6af.jpg)

**Comment by ghost** [comment]
look where it says edited by... look at the last two, bottom is original, one above it is edited.

**Comment by iop3** [comment]
~~AHAHAHHAH WHY IN THE WORLD DO YOU HAVE LIGHT MODE TURNED ON!!~~

the edited by is blocked by the thing.

**Comment by iop3** [comment]
and if you people want to continue talking imma reopen this issue.

**Comment by ghost** [comment]
@TheQuackyDuck thats why i gave you the screenshot
(dark mode needs some more refining)

**Comment by iop3** [comment]
ik but i cant see it.

![image](https://user-images.githubusercontent.com/79815764/125111683-0bf39e80-e09b-11eb-8499-5499cccd74dc.png)

it's blocked by this

![image](https://user-images.githubusercontent.com/79815764/125111720-17df6080-e09b-11eb-8f28-1a1bec785a17.png)

**Comment by ghost** [comment]
@TheQuackyDuck scroll down.

**Comment by iop3** [comment]
![image](https://user-images.githubusercontent.com/79815764/125115255-f0d75d80-e09f-11eb-8af1-80e50bbd0d85.png)

?

**Comment by ghost** [comment]
@TheQuackyDuck the bit highlighted in green is what he added, the bit in red is what he removed.

**Comment by iop3** [comment]
ok so...?

**Comment by ghost** [comment]
> > Apparently, I think it is funny to put a cipher, and watch these gullible idiots just like myself suffer, because I have the sense of humour of a two year old.
> 
> woah

@TheQuackyDuck notice the "woah" after the quote he did, basically trying to frame me for starters.

**Comment by iop3** [comment]
oh

**Comment by whippingdot** [comment]
bruh @firefish111 wut is wrong wit you lol

sorry @StringentDev

**Comment by ghost** [comment]
@whippingdot has he done this before?

**Comment by whippingdot** [comment]
Nope only since you came

**Comment by ghost** [comment]
@whippingdot well, firefish was the one who invited me originally so i dont know whats going through his head

**Comment by whippingdot** [comment]
I don't know why you locked the conversation cause everyone here I am pretty sure is a collaborator...maybe not stringent i don't know but fine

**Comment by firefish111** [comment]
it says precisely why I locked the conversation: `too heated`

**Comment by firefish111** [comment]
> @whippingdot well, firefish was the one who invited me originally so i dont know whats going through his head

I invited you back in April because you asked, you only asked to be invited again come June, and this is becoming the entire Tr1angular thing but worse. Trying to perfect projects which, I and others have been perfecting since Christmas? Here, newcomers are lower down the hierarchy, than people who have been here since the beginning. I'm sorry.

**Comment by firefish111** [comment]
@StringentDev https://github.com/DASM-Project/DASM/issues/18

**Comment by whippingdot** [comment]
hey can he still be in the org without you guys fighting? It is ok if he wants to make the project better - but he just shouldn't add commits randomly and do stuff like that. He did create an issue asking if he could change it...?

**Comment by firefish111** [comment]
> hey can he still be in the org without you guys fighting? It is ok if he wants to make the project better - but he just shouldn't add commits randomly and do stuff like that. He did create an issue asking if he could change it...?

Yeah that's fine, if he gets into the teams he deserves to be in, because he's only learning nodejs, I had to make an entire tutorial to teach him stuff. He also tried to revive a dead project, virra, as DASM. You see, when he asked to be invited the first time, I had invited his old account, @Lethdev2019, and then we got annoyed at each other and he asked to be invited again after I told him about the tr1angular incident.

**Comment by iop3** [comment]
im not a mod, why can i close issues..? or every member of a org can close issues in that org?

**Comment by firefish111** [comment]
ye, these permissions suck, i trieed to fix it with the nidejs team and deleting and recreatnig was the only qway i know how

**Comment by ghost** [comment]
> > hey can he still be in the org without you guys fighting? It is ok if he wants to make the project better - but he just shouldn't add commits randomly and do stuff like that. He did create an issue asking if he could change it...?
> 
> Yeah that's fine, if he gets into the teams he deserves to be in, because he's only learning nodejs, I had to make an entire tutorial to teach him stuff. He also tried to revive a dead project, virra, as DASM. You see, when he asked to be invited the first time, I had invited his old account, @Lethdev2019, and then we got annoyed at each other and he asked to be invited again after I told him about the tr1angular incident.

I was never in the group to begin with as i'd have been in a team by them... and the tutorial came WAAAY after i learnt most of the stuff. To correct that, DASM was temporary, as the name did not quite look nice, It is called Vera, as the "ir" part was bugging me.

Also if you are trying to change perms, you have to go into org settings. Another way could be creating a team with admin perms whilst taking it off the others.

**Comment by ghost** [comment]
Also i tried reviving Virra, which worked successfully as work is being done again. Oh btw, Gcc did not like `printf` so i had to use `puts` instead as it must be a `literal string`

**Comment by ghost** [comment]
> > @whippingdot well, firefish was the one who invited me originally so i dont know whats going through his head
> 
> I invited you back in April because you asked, you only asked to be invited again come June, and this is becoming the entire Tr1angular thing but worse. Trying to perfect projects which, I and others have been perfecting since Christmas? Here, newcomers are lower down the hierarchy, than people who have been here since the beginning. I'm sorry.

It does not mean i will apply it **without permission**. I'm not that bad... I think... Also... I'm pretty sure that tutorial was in `C` not `nodejs` @firefish111 as i commented that it "looks just like javascript" which to be honest it does...

Besides, im waaaay underqualified to go into the `C` team.

**Comment by whippingdot** [comment]
> im not a mod, why can i close issues..? or every member of a org can close issues in that org?

bruh issues are not supposed to be created by the people in the org, they are supposed to be created by external people. The people in the org are supposed to follow up on the issues and close it 🤦

**Comment by whippingdot** [comment]
> > > hey can he still be in the org without you guys fighting? It is ok if he wants to make the project better - but he just shouldn't add commits randomly and do stuff like that. He did create an issue asking if he could change it...?
> > 
> > 
> > Yeah that's fine, if he gets into the teams he deserves to be in, because he's only learning nodejs, I had to make an entire tutorial to teach him stuff. He also tried to revive a dead project, virra, as DASM. You see, when he asked to be invited the first time, I had invited his old account, @Lethdev2019, and then we got annoyed at each other and he asked to be invited again after I told him about the tr1angular incident.
> 
> I was never in the group to begin with as i'd have been in a team by them... and the tutorial came WAAAY after i learnt most of the stuff. To correct that, DASM was temporary, as the name did not quite look nice, It is called Vera, as the "ir" part was bugging me.
> 
> Also if you are trying to change perms, you have to go into org settings. Another way could be creating a team with admin perms whilst taking it off the others.

I never allowed him to change perms though. He isn't telling me what he changed

**Comment by iop3** [comment]
> > im not a mod, why can i close issues..? or every member of a org can close issues in that org?
> 
> bruh issues are not supposed to be created by the people in the org, they are supposed to be created by external people. The people in the org are supposed to follow up on the issues and close it 🤦

im a idiot, thanks for pointing that out @whippingdot

**Comment by ghost** [comment]
> > > > hey can he still be in the org without you guys fighting? It is ok if he wants to make the project better - but he just shouldn't add commits randomly and do stuff like that. He did create an issue asking if he could change it...?
> > > 
> > > 
> > > Yeah that's fine, if he gets into the teams he deserves to be in, because he's only learning nodejs, I had to make an entire tutorial to teach him stuff. He also tried to revive a dead project, virra, as DASM. You see, when he asked to be invited the first time, I had invited his old account, @Lethdev2019, and then we got annoyed at each other and he asked to be invited again after I told him about the tr1angular incident.
> > 
> > 
> > I was never in the group to begin with as i'd have been in a team by them... and the tutorial came WAAAY after i learnt most of the stuff. To correct that, DASM was temporary, as the name did not quite look nice, It is called Vera, as the "ir" part was bugging me.
> > Also if you are trying to change perms, you have to go into org settings. Another way could be creating a team with admin perms whilst taking it off the others.
> 
> I never allowed him to change perms though. He isn't telling me what he changed

🤷‍♂️ are you sure you did not allow that?

**Comment by firefish111** [comment]
> Another way could be creating a team with admin perms whilst taking it off the others.

well I was experimenting basically, found that the team has admin perms, recreated without, and now I think we should do the same for every team

**Comment by whippingdot** [comment]
> > > > > hey can he still be in the org without you guys fighting? It is ok if he wants to make the project better - but he just shouldn't add commits randomly and do stuff like that. He did create an issue asking if he could change it...?
> > > > 
> > > > 
> > > > Yeah that's fine, if he gets into the teams he deserves to be in, because he's only learning nodejs, I had to make an entire tutorial to teach him stuff. He also tried to revive a dead project, virra, as DASM. You see, when he asked to be invited the first time, I had invited his old account, @Lethdev2019, and then we got annoyed at each other and he asked to be invited again after I told him about the tr1angular incident.
> > > 
> > > 
> > > I was never in the group to begin with as i'd have been in a team by them... and the tutorial came WAAAY after i learnt most of the stuff. To correct that, DASM was temporary, as the name did not quite look nice, It is called Vera, as the "ir" part was bugging me.
> > > Also if you are trying to change perms, you have to go into org settings. Another way could be creating a team with admin perms whilst taking it off the others.
> > 
> > 
> > I never allowed him to change perms though. He isn't telling me what he changed
> 
> 🤷‍♂️ are you sure you did not allow that?

No i mean like i didn't allow him verbally - he has the perms to do it

**Comment by whippingdot** [comment]
I made him a mod @StringentDev

**Comment by whippingdot** [comment]
> > Another way could be creating a team with admin perms whilst taking it off the others.
> 
> well I was experimenting basically, found that the team has admin perms, recreated without, and now I think we should do the same for every team

WHAT DO YOU MEAN BY ADMIN PERMS PLEASE EXPLAIN

If it means that they can edit any repository in that team, then I think you should leave it that way cause I want contributions from everyone without having to manually invite 30 people...

**Comment by firefish111** [comment]
> > > Another way could be creating a team with admin perms whilst taking it off the others.
> > 
> > 
> > well I was experimenting basically, found that the team has admin perms, recreated without, and now I think we should do the same for every team
> 
> WHAT DO YOU MEAN BY ADMIN PERMS PLEASE EXPLAIN
> 
> If it means that they can edit any repository in that team, then I think you should leave it that way cause I want contributions from everyone without having to manually invite 30 people...

there are things called pull requests

**Comment by ghost** [comment]
> > > > Another way could be creating a team with admin perms whilst taking it off the others.
> > > 
> > > 
> > > well I was experimenting basically, found that the team has admin perms, recreated without, and now I think we should do the same for every team
> > 
> > 
> > WHAT DO YOU MEAN BY ADMIN PERMS PLEASE EXPLAIN
> > If it means that they can edit any repository in that team, then I think you should leave it that way cause I want contributions from everyone without having to manually invite 30 people...
> 
> there are things called pull requests

that is true... and is a pretty good idea... BUT I am talking about setting write perms to read perms. You can also set individual access to repos!

**Comment by firefish111** [comment]
> > > > > Another way could be creating a team with admin perms whilst taking it off the others.
> > > > 
> > > > 
> > > > well I was experimenting basically, found that the team has admin perms, recreated without, and now I think we should do the same for every team
> > > 
> > > 
> > > WHAT DO YOU MEAN BY ADMIN PERMS PLEASE EXPLAIN
> > > If it means that they can edit any repository in that team, then I think you should leave it that way cause I want contributions from everyone without having to manually invite 30 people...
> > 
> > 
> > there are things called pull requests
> 
> that is true... and is a pretty good idea... BUT I am talking about setting write perms to read perms. You can also set individual access to repos!

that's a lot of quotes

**Comment by whippingdot** [comment]
> > > > > Another way could be creating a team with admin perms whilst taking it off the others.
> > > > 
> > > > 
> > > > well I was experimenting basically, found that the team has admin perms, recreated without, and now I think we should do the same for every team
> > > 
> > > 
> > > WHAT DO YOU MEAN BY ADMIN PERMS PLEASE EXPLAIN
> > > If it means that they can edit any repository in that team, then I think you should leave it that way cause I want contributions from everyone without having to manually invite 30 people...
> > 
> > 
> > there are things called pull requests
> 
> that is true... and is a pretty good idea... BUT I am talking about setting write perms to read perms. You can also set individual access to repos!

It would take a long time...but if a lot of people really think so...

> > > > > > Another way could be creating a team with admin perms whilst taking it off the others.
> > > > > 
> > > > > 
> > > > > well I was experimenting basically, found that the team has admin perms, recreated without, and now I think we should do the same for every team
> > > > 
> > > > 
> > > > WHAT DO YOU MEAN BY ADMIN PERMS PLEASE EXPLAIN
> > > > If it means that they can edit any repository in that team, then I think you should leave it that way cause I want contributions from everyone without having to manually invite 30 people...
> > > 
> > > 
> > > there are things called pull requests
> > 
> > 
> > that is true... and is a pretty good idea... BUT I am talking about setting write perms to read perms. You can also set individual access to repos!
> 
> that's a lot of quotes

Replying to your comment before this one - I know they are called PRs, but I am talking about repos in teams

**Comment by firefish111** [comment]
yeah everybody pull requests each commit, nobody except admins have access to commiting to master. THATS HOW ITS SUPPOSED TO WORK

**Comment by whippingdot** [comment]
i replied somewhere else

**Comment by firefish111** [comment]
very helpful

**Comment by ghost** [comment]
Why do I look at this comment chain and cringe at myself?

###### I do that a lot anyways so no change there

**Comment by zghwi** [comment]
ded

**Comment by firefish111** [comment]
uncle jesus' casino

**Comment by whippingdot** [comment]
> ded

I agree. I can try to revive but my life just isn't as free as before. I don't want to have the responsibility of checking in each day as I also want to improve my coding skills
